### ISSUE TITLE
[MCHECKSTYLE-326] Running mvn install works first time second it does not

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-checkstyle-plugin#617]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** opened **[MCHECKSTYLE-326](https://issues.apache.org/jira/browse/MCHECKSTYLE-326?redirect=false)** and commented

If I try to install the state of maven-shared/maven-shared-utils (r1753303) I can run the following: `mvn clean install` without any issue. If i run it a second time by using `mvn install` than the build will fail with a checksyle error.

First call `mvn clean install`

```
...
[INFO]
[INFO] --- maven-checkstyle-plugin:2.15:check (checkstyle-check) @ maven-shared-utils ---
[INFO] There are 6 errors reported by Checkstyle 6.1.1 with config/maven_checks.xml ruleset.
[INFO] Ignored 6 errors, 0 violation remaining.
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ maven-shared-utils ---
[INFO] Installing /Users/kama/apache-maven/maven-shared/maven-shared-utils/target/maven-shared-utils-3.1.0-SNAPSHOT.jar to /Users/kama/.m2/repository/org/apache/maven/shared/maven-shared-utils/3.1.0-SNAPSHOT/maven-shared-utils-3.1.0-SNAPSHOT.jar
[INFO] Installing /Users/kama/apache-maven/maven-shared/maven-shared-utils/pom.xml to /Users/kama/.m2/repository/org/apache/maven/shared/maven-shared-utils/3.1.0-SNAPSHOT/maven-shared-utils-3.1.0-SNAPSHOT.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.719 s
[INFO] Finished at: 2016-07-18T21:16:13+02:00
[INFO] Final Memory: 33M/710M
[INFO] ------------------------------------------------------------------------
```

Now the second call `mvn install`

```
[INFO]
[INFO] --- maven-jar-plugin:3.0.0:jar (default-jar) @ maven-shared-utils ---
[INFO] Building jar: /Users/kama/apache-maven/maven-shared/maven-shared-utils/target/maven-shared-utils-3.1.0-SNAPSHOT.jar
[INFO]
[INFO] --- maven-site-plugin:3.5.1:attach-descriptor (attach-descriptor) @ maven-shared-utils ---
[INFO]
[INFO] --- maven-checkstyle-plugin:2.15:check (checkstyle-check) @ maven-shared-utils ---
[INFO] There are 7 errors reported by Checkstyle 6.1.1 with config/maven_checks.xml ruleset.
[ERROR] src/main/java/org/apache/maven/shared/utils/logging/package-info.java[1] (header) RegexpHeader: Line does not match expected header line of '^package '.
[INFO] Ignored 6 errors, 1 violation remaining.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.756 s
[INFO] Finished at: 2016-07-18T21:16:44+02:00
[INFO] Final Memory: 37M/708M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.15:check (checkstyle-check) on project maven-shared-utils: You have 1 Checkstyle violation. -> [Help 1]
```

Only if I run with `mvn clean ...` it works..But if I don't run with `clean` it will fail....

---

**Affects:** 2.15

**Issue Links:**
- [MCHECKSTYLE-327](https://issues.apache.org/jira/browse/MCHECKSTYLE-327) Upgrading from 2.15 to 2.17 will fail

**Remote Links:**
- [GitHub Pull Request #26
](https://github.com/apache/maven-shared/pull/26)

**Comment by jira-importer** [comment]
**[Sylwester Lachiewicz](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slachiewicz)** commented

As a workaround - add property _<checkstyle.excludes>**/package-info.java</checkstyle.excludes>_

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

Hi Sylwester, unfortunately the package-info.java do not exist anymore in the current state..

**Comment by jira-importer** [comment]
**[Guillaume Boué](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gboue)** commented

I can't reproduce this with latest 3.0.1 of the plugin (tested by building maven-shared-utils 3.1.0). Is this still an issue for you?

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

If you can't reproduce it it is no longer an issue...Just close it...

**Comment by jira-importer** [comment]
**[Guillaume Boué](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gboue)** commented

All right, considering this fixed in 3.0.1.
